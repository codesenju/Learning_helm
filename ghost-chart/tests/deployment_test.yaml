templates:
  - deployment.yaml
tests:
  # - it: pod spec should match snapshot
  #   asserts:
  #     - matchSnapshot:
  #         path: spec.template.spec
  # or you can snapshot the whole manifest
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: {}