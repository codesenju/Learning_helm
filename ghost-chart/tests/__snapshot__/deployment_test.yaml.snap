manifest should match snapshot:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app: ghost
      name: ghost
    spec:
      replicas: 1
      selector:
        matchLabels:
          app: ghost
      template:
        metadata:
          labels:
            app: ghost
        spec:
          containers:
            - env:
                - name: NODE_ENV
                  value: development
              image: nginx:alpine
              name: ghost
              ports:
                - containerPort: 80
                  name: http
          restartPolicy: Always
